graphjet-core/src/test/java/com/twitter/graphjet/algorithms/counting/TopSecondDegreeByCountTest.java

```diff
@@ -159,7 +164,7 @@ public void testTopSecondDegreeByCountWithSocialProofTypeUnionsWithSmallGraph()
       new RequestedSetFilter(new NullStatsReceiver())
     ));
 
-    TopSecondDegreeByCountRequest topSecondDegreeByCountRequest = new TopSecondDegreeByCountRequest(
+    TopSecondDegreeByCountRequestForTweet topSecondDegreeByCountRequestForTweet = new TopSecondDegreeByCountRequestForTweet(
```

Comment:
We should add checkstyle to the project to catch these - I think we've probably introduced some style issues, but I can take a crack at cleaning it up..